### PR TITLE
Add formal definition of  workloads

### LINKED ISSUE #11285: Include a formal definition of "Workloads" in k8s.io/docs/concepts/workloads/
**This is a...** 

- [ ] Feature Request
- [ x] Bug Report

**Problem:**
Im considering this a sort of bug. Could we have a more formal definition of what a k8s workload is? Maybe on the Workloads page? The k8s and gcp documentation seems kind of ambiguous here and the gcp console has an actual "Workloads" section, but im not 100% sure all the stuff i will see in that UI pane. "Workloads" arent an actual k8s construct i guess, there is no workloads class or anything but its like a way of categorizing specific k8s objects? This isnt clear from the docs

**Proposed Solution:**
In the workloads page have a paragraph or two that starts with "Kubernetes workloads are...". This will be helpful for new users trying to wrap their head around k8s.

**Page to Update:**
https://kubernetes.io/docs/concepts/workloads/

### PR DESCRIPTION
**Pull request by RA489**
Adding the formal definition of  workloads
fixes #11285

### CODE REVIEW

**Comment by zparnold** [comment]
@RA489 What do you think of @Bradamant3 's suggestions? We appreciate the contribution and definitely want to include it!

**Comment by RA489** [comment]
@zparnold sure, i will include it

**Comment by RA489** [comment]
@Bradamant3 @zparnold updated as per suggestion. PTAL

**Comment by Rajakavitha1** [comment]
@Bradamant3 could you please take a look at the updates made to this PR.

**Comment by RA489** [comment]
@zparnold @Bradamant3 could you please review this PR.

File: content/en/docs/reference/glossary/workload.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
 ---
-title: Workload
-id: workload
-date: 2019-02-12
-full_link: /docs/concepts/workloads/
+title: Workloads
+id: workloads
+date: 2019-02-13
+full_link: https://kubernetes.io/docs/concepts/workloads/
```
Comment (Bradamant3) [inline content/en/docs/reference/glossary/workload.md]: the full link should still be the relative path to the docs (in other words, revert this change). See other glossary examples.

Comment (Bradamant3) [inline content/en/docs/reference/glossary/workload.md]: full link should still be the relative path to the /docs directory. IOW, revert this change please

File: content/en/docs/reference/glossary/workload.md [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
 short_description: >
-  A set of applications for processing information to serve a purpose that is valuable to a single user or group of users.
+   A workload is a framing term to refer to a set of APIs/resources/objects.
```
Comment (Bradamant3) [inline content/en/docs/reference/glossary/workload.md]: Sorry, I used "framing term" as a way of redirecting the conversation. It wasn't meant as the language for the glossary entry. See the API refdocs: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#
And you might want to explore the language of workloads elsewhere in the docs a bit further too.

File: content/en/docs/reference/glossary/workload.md [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
-aka:
+aka: 
 tags:
+- fundamental
+- core-object
 - workload
 ---
-A workload consists of a system of services or applications that can run to fulfill a
-task or carry out a business process.
+  A workload is a framing term to refer to a set of APIs/resources/objects.
```
Comment (Bradamant3) [inline content/en/docs/reference/glossary/workload.md]: See previous comment

File: content/en/docs/reference/glossary/workload.md [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
 
-<!--more-->
+<!--more--> 
 
-Alongside the computer code that runs to carry out the task, a workload also entails
-the infrastructure resources that actually run that code.
+Workload is a framing term to refer to a set of APIs/resources/objects. Kubernetes performs the 
+deployment and updates the workload with the current state of the application. Workloads let you define the 
+rules for application scheduling, scaling, and upgrade.
+Workloads includes the Daemon Sets, Deployments, Jobs, Pods, Replica Sets, Replication Controllers, Stateful Sets.
```
Comment (Bradamant3) [inline content/en/docs/reference/glossary/workload.md]: s/includes/include
s/Daemon Sets/DaemonSet
s/Replica Sets/ReplicaSet
s/Replication Controllers/ReplicationController
s/Stateful Sets/StatefulSet

If you're going this far, suggest adding the rest of the list. Or remove the list altogether.

It's also really not true that workload objects define rules for application scheduling, etc ... yes, scheduling happens to workload objects, but this wording is misleading. (Same holds true for scaling and upgrade)

**Comment by RA489** [comment]
@Bradamant3 thanks for the review.

File: content/en/docs/reference/glossary/workload.md (hunk 1 above)
Comment (RA489) [reply content/en/docs/reference/glossary/workload.md]: reverted

File: content/en/docs/reference/glossary/workload.md (hunk 2 above)
Comment (RA489) [reply content/en/docs/reference/glossary/workload.md]: done

File: content/en/docs/reference/glossary/workload.md (hunk 4 above)
Comment (RA489) [reply content/en/docs/reference/glossary/workload.md]: done

**Comment by RA489** [comment]
@Bradamant3 PTAL.
